caravel/assets/visualizations/heatmap.js

```diff
@@ -10,8 +10,11 @@ require('./heatmap.css');
 // https://jsfiddle.net/cyril123/h0reyumq/
 function heatmapVis(slice) {
   function refresh() {
+    // Header for panel in explore v2
+    const header = document.getElementsByClassName('panel-title');
+    const headerHeight = header ? header[0].offsetHeight : 0;
```

Comment:
i would use `getBoundingClientRect` here, it has better browser support compared to `offsetHeight`
https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetHeight

can you confirm that this change doesn't affect the current explore view too much?